### ISSUE TITLE
Inquiries about DeblurDiNATL training details

### ISSUE THREAD

**Issue by xinran27** [HanzhouLiu/DeblurDiNAT#5]
First of all, thank you for your great work. I'm studying your model and am particularly interested in the implementation details. In order to better understand the training process of the model and the resources required, I have the following questions:

1. How many GPU cards did you use when training the DeblurDiNAT model?
2. How long does a complete training cycle take approximately?

Knowing this information will be of great help to my own research efforts. I look forward to hearing from you and thank you again for your contribution.

**Comment by HanzhouLiu** [comment]
Thanks for your attention to our work.
1. I used eight A100 GPUs to train the model. The batch size was set to 8. If you use a larger/smaller batch size, you might want to tune the learning rate.
2. The first training phase took around 5 - 6 days; the second training phase took around 3 - 4 days. 

The training details should be clarified in the following preprint paper. 
[Paper](https://arxiv.org/abs/2403.13163)

**Comment by xinran27** [comment]
Thank you very much for answering my questions promptly.
